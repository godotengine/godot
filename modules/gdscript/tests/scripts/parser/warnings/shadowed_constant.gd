# See also `parser-errors/redefine-class-constant.gd`.
const TEST = 25


func test():
	# Warning here. This is not an error because a new constant is created,
	# rather than attempting to set the value of an existing constant.
	const TEST = 50
