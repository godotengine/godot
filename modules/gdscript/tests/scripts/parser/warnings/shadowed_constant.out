GDTEST_OK
>> WARNING
>> Line: 8
>> UNUSED_LOCAL_CONSTANT
>> The local constant "TEST" is declared but never used in the block. If this is intended, prefix it with an underscore: "_TEST".
>> WARNING
>> Line: 8
>> SHADOWED_VARIABLE
>> The local constant "TEST" is shadowing an already-declared constant at line 2 in the current class.
